### ISSUE TITLE
Tool capabilities

### ISSUE THREAD

**Issue by psimk** [mediamonks/porter#5]
Currently the tools provide 2 sub commands
* `dev` - start the development server
* `build` - create a production build

What other sub commands should the tool provide?
perhaps:
* `serve` - start a local server for a production build

What options should these sub commands allow?
* `build --parallel` - allow building multiple configs in parallel

What and how should be made customizable to the end user?
Currently we allow defining a custom `webpack.config.ts`

**Comment by psimk** [comment]
tools will be folded into skeletons, so this is no longer valid
